src/components/ButtonPanel.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Button from './Button';
+
+export default function ButtonPanel() {
+  return (
+    <>
+      <div>
+        <Button name="AC" />
+        <Button name="+/-" />
+        <Button name="%" />
+        <Button name="+" />
+      </div>
+      <div>
+        <Button name="7" />
+        <Button name="8" />
+        <Button name="9" />
+        <Button name="X" />
+      </div>
+      <div>
+        <Button name="4" />
+        <Button name="5" />
+        <Button name="6" />
+        <Button name="-" />
+      </div>
+      <div>
+        <Button name="1" />
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+    </>
```

Comment:
- You missed adding the last group of operators:
```react
<div>
    <Button name="0" />
    <Button name="." />
    <Button name="=" />
 </div>
```

Reply:
**Indeed, Thanks:+1:**
Done